app/assets/javascript/pageflow/linkmap_page/editor/models/generated_file.js

```diff
@@ -1,4 +1,4 @@
-pageflow.linkmapPage.ProcessedFile = pageflow.UploadedFile.extend({
+pageflow.linkmapPage.GeneratedFile = pageflow.ReusableFile.extend({
```

Comment:
Maybe we can take the opportunity to also call this `GeneratedImageFile` to match the rails model name?